packages/meditrak-server/src/kobo/startSyncWithKoBo.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import keyBy from 'lodash.keyby';
+import winston from 'winston';
+import { DataBroker } from '@tupaia/data-broker';
+import { generateId } from '@tupaia/database';
+import { reformatDateStringWithoutTz } from '@tupaia/utils';
+
+const PERIOD_BETWEEN_SYNCS = 10 * 60 * 1000; // 10 minutes between syncs
+const SERVICE_NAME = 'KoBo';
+
+export async function startSyncWithKoBo(models) {
+  if (process.env.KOBO_SYNC_DISABLE) {
+    // eslint-disable-next-line no-console
+    console.log('KoBo sync is disabled');
+  } else {
+    const dataBroker = new DataBroker();
+    setInterval(() => syncWithKoBo(models, dataBroker), PERIOD_BETWEEN_SYNCS);
```

Comment:
```suggestion
    const koboSyncServices = await models.syncService.find({ type: SERVICE_TYPE });
    koboSyncServices.forEach(service => setInterval(() => syncWithKoBo(models, dataBroker, service.code), PERIOD_BETWEEN_SYNCS));
```
This also requires something I didn't think of in the other PR - a `sync_service.type` column